src/System.ServiceModel.Http/tests/ServiceModel/DnsEndpointEntityTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
NetTcp is probably a better place as the E2E scenario we are enabling is a NetTcp scenario. Are there any other suggestions?